app/src/main/kotlin/com/wire/android/ui/userprofile/UserProfileViewModel.kt

```diff
@@ -44,12 +44,12 @@ class UserProfileViewModel @Inject constructor(
             getSelf().collect {
                 userProfileState = SelfUserProfileState(
                     status = UserStatus.AVAILABLE,
-                    fullName = it.name!!,
-                    userName = it.handle!!,
+                    fullName = it.name ?: "",
+                    userName = it.handle ?: "",
```

Comment:
Not a real suggestion for improvement, as I don't have any preference and we don't have a standard for this in the team. But just FYI:

Kotlin allows us to call functions on optionals. So things like `myString.orEmpty()` or `myString.isNullOrEmpty()` exist in the stdlib.

They are defined as extension functions of nullable types:

```kotlin
fun String?.isNullOrEmpty(): Boolean = this == null || this.isEmpty()

fun String.orEmpty(): String = this ?: ""
```

I guess we could do the same on swift:

```swift
extension Optional where Wrapped: String {
    var isNilOrEmpty: Bool {
        return self?.isEmpty ?? true
    }
}
```

Having two ways of doing the same thing is not always nice, but 🤷🏼‍♂️
It exists anyway.